apis/fluent-en_GB/atrium-api-fluent-en_GB/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableSubjectChangersSamples.kt

```diff
@@ -0,0 +1,39 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.asList
+import ch.tutteli.atrium.api.fluent.en_GB.toContain
+import ch.tutteli.atrium.api.fluent.en_GB.toEqual
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class IterableSubjectChangersSamples {
+    @Test
+    fun asListFeature() {
+        expect(0..2)
+            .asList()
+            .toEqual(listOf(0, 1, 2))
+
+        fails {
+            expect(0..2)
+                .asList()
+                .toContain(3)
+                .toContain(4)
```

Comment:
```suggestion
                .asList()  // subject is now of type List<String>
                .toContain(3)  // fails
                .toContain(4)  // not evaluated/reported because above `toContain` already fails
          //                      use `.asList { ... }` if you want that all expectations are evaluated
```